Content.Server/Labels/Label/Components/LabelComponent.cs

```diff
@@ -1,12 +1,22 @@
 namespace Content.Server.Labels.Components
 {
+    /// <summary>
+    /// Gives opportunity to add labels by entities with the <see cref="HandLabelerComponent"/>
+    /// </summary>
     [RegisterComponent]
     public sealed class LabelComponent : Component
     {
+        /// <summary>
+        /// Label text. If you using that field in your prototype, you must set originalName field too, and set name of entity like it with label
+        /// </summary>
```

Comment:
`The actual text in the label.`* (the remarks that follows isnt necessary because of the next review)